scripts/deploy-wc.sh

```diff
@@ -27,12 +27,26 @@ kubectl -n fluentd create secret generic opensearch \
 # Add example resources.
 # We use `create` here instead of `apply` to avoid overwriting any changes the
 # user may have done.
-kubectl create -f "${SCRIPTS_PATH}/../manifests/examples/fluentd/fluentd-extra-config.yaml" \
-    2>/dev/null || echo "fluentd-extra-config configmap already in place. Ignoring."
-kubectl create -f "${SCRIPTS_PATH}/../manifests/examples/fluentd/fluentd-extra-plugins.yaml" \
-    2>/dev/null || echo "fluentd-extra-plugins configmap already in place. Ignoring." >&2
-kubectl create -f "${SCRIPTS_PATH}/../helmfile/charts/manifests/user-rbac/clusterrolebindings/extra-user-view.yaml" \
-    2>/dev/null || echo "extra-user-view ClusterRoleBinding already in place. Ignoring." >&2
+if [ "$(kubectl get configmap fluentd-extra-config -n fluentd)" ] ; then
+  echo "fluentd-extra-config ConfigMap already in place. Ignoring."
+else
+  echo "Creating fluentd-extra-config ConfigMap"
+  kubectl create -f "${SCRIPTS_PATH}/../manifests/examples/fluentd/fluentd-extra-config.yaml"
+fi
+
+if [ "$(kubectl get configmap fluentd-extra-plugins -n fluentd)" ] ; then
+  echo "fluentd-extra-plugins ConfigMap already in place. Ignoring."
+else
+  echo "Creating fluentd-extra-plugins ConfigMap"
+  kubectl create -f "${SCRIPTS_PATH}/../manifests/examples/fluentd/fluentd-extra-plugins.yaml"
```

Comment:
Should these really be in `manifests/examples`? Feels strange that we apply configs that are examples. Should they maybe be part of the `CK8S_CONFIG_PATH/manifests/fluentd`? Because these are configs that should be configured by the user, if i understand it correctly

Reply:
@viktor-f

Reply:
I would say keep them where they are, they do contain actual examples so i think it is fine where they are now.